addons/account/models/account_report.py

```diff
@@ -342,7 +342,7 @@ def _validate_groupby_no_child(self):
             if report_line.parent_id.groupby or report_line.parent_id.user_groupby:
                 raise ValidationError(_("A line cannot have both children and a groupby value (line '%s').", report_line.parent_id.name))
 
-    @api.constrains('expression_ids', 'groupby')
+    @api.constrains('expression_ids', 'user_groupby')
```

Comment:
```suggestion
    @api.constrains('expression_ids', 'groupby', 'user_groupby')
```
Shouldn't we constrain both ?